actionpack/lib/action_controller/metal/force_ssl.rb

```diff
@@ -4,17 +4,10 @@
 require "active_support/core_ext/hash/slice"
 
 module ActionController
-  # This module provides a method which will redirect the browser to use the secured HTTPS
-  # protocol. This will ensure that users' sensitive information will be
-  # transferred safely over the internet. You _should_ always force the browser
-  # to use HTTPS when you're transferring sensitive information such as
-  # user authentication, account information, or credit card information.
+  # This module is deprecated in favor of +config.force_ssl+ in your environment
+  # config file. This will ensure all communication to non-whitelisted endpoints
+  # served by your application occurs over HTTPS.
```

Comment:
I think you want to add `# :stopdoc:` (or `# :nodoc:`? I can't remember exactly) to remove this module from documentation altogether. If you just remove the documentation, the class will be listed on there but just with no documentation.

Reply:
I  believe it's ` # :nodoc:` - the former is for when you want to stop public methods or other documentation elements being added to the current documentation container until it's started again or the file ends.